include/alpaka/platform/PlatformCpuSycl.hpp

```diff
@@ -18,27 +18,20 @@ namespace alpaka
 {
     namespace detail
     {
-        struct SyclCpuSelector
+        template<>
+        struct SYCLDeviceSelector<TagCpuSycl>
         {
             auto operator()(sycl::device const& dev) const -> int
             {
                 return dev.is_cpu() ? 1 : -1;
             }
+
+            static constexpr char name[] = "CpuSycl";
```

Comment:
If this is only used in the accelerator specialisations, should it be moved there ?
Or, if it is used anywhere else, maybe it should be moved to the platform classes ?

Reply:
It's used only [here](https://github.com/AuroraPerego/alpaka/blob/SYCLrefactoring/include/alpaka/acc/AccGenericSycl.hpp#L167), so I can try to move it.